user-image/connectors/neuro.bash

```diff
@@ -5,7 +5,8 @@ set -euo pipefail
 jupyter serverextension enable  --sys-prefix --py nbserverproxy
 
 # pycortex seems to need cython, but isn't explicitly defined as a dependency
-pip install --no-cache-dir cython==0.26
+# pip install --no-cache-dir cython==0.26
+# cython 0.25.2 installed in Dockerfile		--Gunjan
```

Comment:
Don't leave in commented out code, remove it. And add a comment in where cython is being installed that it's used for multiple classes.